core-tests/src/test/java/org/mule/runtime/core/internal/el/ExpressionLanguageExtensionTestCase.java

```diff
@@ -26,16 +27,15 @@
 import org.mule.runtime.core.internal.message.InternalMessage;
 import org.mule.runtime.core.privileged.registry.RegistrationException;
 
+import org.junit.Test;
+
 import java.text.DateFormat;
 import java.util.Calendar;
-import java.util.HashMap;
 import java.util.Map;
 
-import org.junit.Test;
-
 public class ExpressionLanguageExtensionTestCase extends AbstractELTestCase {
 
-  private final String a = "hi";
+  private String a = "hi";
```

Comment:
why remove final? If it is not final as b, maybe it shouldn't be declared as an instance field but as a local variable in the test. Actually if it is something to compare to, this should be a constant.